app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,62 @@
+$header-font-color:#088da5;
+$font-color: #20b2aa;
+
+body {
+
+background-color: black;
+
+}
+
+h1, h3 {
+  text-align: center;
+  color: $header-font-color;
+
+}
+
+
+#about {
+
+border: 3px solid black;
+text-align: center;
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Properties should be ordered border, color, margin, padding, text-align